examples/ecommerce-netlify-functions/functions/checkout.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+const stripe = require('stripe')(process.env.STRIPE_KEY);
+
+const { Cart, Order, Product } = require('../models');
+
+const handler = async(event) => {
+  try {
+    const stripeProducts = { line_items: [] };
+    const total = 0;
+    for (let i = 0; i < event.body.products.length; i++) {
+      const product = await Product.findOne({_id: event.body.products[i].productId});
+      stripeProducts.line_items.push({
+        price_data: {
+          currency: 'usd',
+          product_data: {
+            name: product.productName
+          },
+          unit_amount: product.productPrice,
+        },
+        quantity: event.body.products[i].quantity
+      });
+      total = total + (product.productPrice * event.body.products[i].quantity);
+    }
+    const salesTax = total * .06;
```

Comment:
Let's get rid of `salesTax` throughout this codebase. It isn't necessary for this example.